pages/1.13/cli/command-reference/dcos-node/dcos-node-list/index.md

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
This page doesn't need to be created in this PR, we're already creating it in https://github.com/mesosphere/dcos-docs-site/pull/2107 as mentioned in https://jira.mesosphere.com/browse/DCOS-50125?focusedCommentId=253016&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-253016